test/test_quantized.py

```diff
@@ -147,25 +147,30 @@ def test_qadd_scalar_relu(self, A, b):
         add_scalar_relu = torch.ops.quantized.add_scalar_relu
 
         A, (scale, zero_point, dtype) = A
+        check_val = zero_point + b
+        if dtype == torch.qint8 and check_val < -128 or check_val > 127:
+            return
+        if dtype == torch.quint8 and check_val > 255 or check_val < 0:
+            return
+        if dtype == torch.quint8 and check_val > 2**31 - 1 or check_val < -(2**31):
+            return
         A = A.astype(np.float32)
         qA = torch.quantize_linear(torch.from_numpy(A), scale, zero_point, dtype)
 
         C = qA.dequantize() + b
         C_relu = copy.deepcopy(C)
         C_relu[C_relu < 0] = 0
 
-        C_ref = torch.quantize_linear(C, scale, zero_point, dtype)
-        C_relu_ref = torch.quantize_linear(C_relu, scale, zero_point, dtype)
+        C_hat = add_scalar(qA, b)
+        C_relu_hat = add_scalar_relu(qA, b)
 
-        C_hat = add_scalar(qA, b, scale=scale, zero_point=zero_point)
-        C_relu_hat = add_scalar_relu(qA, b, scale=scale, zero_point=zero_point)
-
-        self.assertEqual(C_ref, C_hat,
+        self.assertEqual(C, C_hat.dequantize(), prec=1.0,
```

Comment:
Can we tighten the prec value? Does this still need to be 1?

Reply:
This fails for any arbitrary precision value